corporate/views/support.py

```diff
@@ -312,48 +312,44 @@ def get_realm_plan_type_options_for_discount() -> List[SupportSelectOption]:
     return plan_types
 
 
-VALID_MODIFY_PLAN_METHODS = [
-    "downgrade_at_billing_cycle_end",
-    "downgrade_now_without_additional_licenses",
-    "downgrade_now_void_open_invoices",
-    "upgrade_plan_tier",
-]
-
-VALID_STATUS_VALUES = [
-    "active",
-    "deactivated",
-]
-
-VALID_BILLING_MODALITY_VALUES = [
-    "send_invoice",
-    "charge_automatically",
-]
```

Comment:
Since these are used in two functions, I think it's still worth defining variables with these names (now with `Literal`) and using them in both functions.